cmd/export_trades.go

```diff
@@ -29,17 +32,34 @@ var tradesCmd = &cobra.Command{
 			cmdLogger.Fatal("could not read trades: ", err)
 		}
 
+		failures := 0
 		for _, tradeInput := range trades {
 			trades, err := transform.TransformTrade(tradeInput.OperationIndex, tradeInput.OperationHistoryID, tradeInput.Transaction, tradeInput.CloseTime)
 			if err != nil {
-				cmdLogger.Fatal("could not transform trade ", err)
+				parsedID := toid.Parse(tradeInput.OperationHistoryID)
+				locationString := fmt.Sprintf("from ledger %d, transaction %d, operation %d", parsedID.LedgerSequence, parsedID.TransactionOrder, parsedID.OperationOrder)
```

Comment:
Very clean representation. Just FYI - you could potentially use "%v+" - but I think this is cleaner.